benefits/oauth/client.py

```diff
@@ -1,36 +1,52 @@
-import logging
+"""
+The oauth application: helpers for working with OAuth clients.
+"""
 
-from django.conf import settings
+import logging
 
 from authlib.integrations.django_client import OAuth
 
+from benefits.core.models import AuthProvider
+
 
 logger = logging.getLogger(__name__)
 
+oauth = OAuth()
 
-_OPENID_SCOPE = "openid"
 
+def client_kwargs(scope=None):
```

Comment:
Would this function ever be used outside of this module? If not, I suggest prefixing with a `_`

Reply:
You're right, it probably would not! 👍